.github/workflows/code-review.yml

```
@@ -0,0 +1,41 @@
+          echo "Comment: ${{ github.event.comment.body }}"
+          echo "Issue Number: ${{ github.event.issue.number }}"
+          echo "Repository: ${{ github.repository }}"
+
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get PR Details
+        id: pr
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:23: Double quote to prevent globbing and word splitting

(shellcheck)